snap/integrity/integrity_test.go

```diff
@@ -227,19 +229,19 @@ case "$1" in
 		exit 0
 		;;
 	format)
-		cp %[1]s %[1]s.verity
-		echo "VERITY header information for %[1]s.verity"
+		truncate -s %[1]d %[2]s.verity
```

Comment:
It looks like the previous version of this test worked because the snap happened to be the same size as the expected hash file. With the updated minimum snap size, it looks like hash files are now at least 8192 bytes. I couldn't figure out any actual function to consistently go from snap size -> hash size, but that'd be better than this.

Reply:
This looks like how it is calculated in verity: https://github.com/nelenkov/cryptsetup/blob/master/lib/verity/verity_hash.c#L285-L301, stored in `hash_device_size`. Probably not worth recreating here.